spec/api/default_api_spec.rb

```diff
@@ -1719,4 +1719,509 @@
     end
   end
 
+  # unit tests for get_next_page get_ten_dlc_sms_brands
+  # Get list of SMS 10DLC Brands
+  # @param [Hash] opts the optional parameters
+  # @return [SMSTenDLCBrandsListResult]
+  describe 'get_ten_dlc_sms_brands_get_next_page test' do
+    it 'should work' do
+      
+      
+      result = @api_instance.get_ten_dlc_sms_brands(
+        
+        {
+          
+        }
+      )
+      result.next_page_uri = "/Accounts/{accountId}/Messages/10DLC/Brands?cursor=1"
+      next_page_result = @api_instance.get_next_page(result)
+      expect(next_page_result).to be_a Freeclimb::SMSTenDLCBrandsListResult
+      # assertion here. ref: https://www.relishapp.com/rspec/rspec-expectations/docs/built-in-matchers
```

Comment:
Do we need this line since we are doing assertions with the expect statement above?

Reply:
i left the line to be consistent with all the other unit tests in the file, they have that comment after using the expect matchers